### ISSUE TITLE
Looking for new episodes promts user twice

### ISSUE THREAD

**Issue by ScumbagDog** [y20k/escapepod#75]
**Describe the bug**
When on the podcast overview, it is possible to update by swiping down, but when doing so, a box pops up to confirm whether the user actually wants to update the podcast list. This seems redundant, as the user has already used a swipe to signify, that they want to update the podcast list manually.

**To Reproduce**
1. Be on the main podcast overview
2. Swipe down to look for new podcast episodes.
3. User is asked whether they want to update episodes via a pop-up box.

**Expected behavior**
In step 3, instead of being asked whether I want to update or not, just look for new episodes as if I had pressed the update button in the popup box.

**Device:**
 - Device: Samsung Galaxy a42
 - OS: Android 11
 - Version Escapepod v1.0.4

**Additional context**
First of all, I absolutely love the app, it works great.
But it has been bothering me when I swipe down, that I get asked if I'm sure I want to update.
If I want to manually update podcasts, this is a redundant prompt, because I have already declared my intention by swiping down, and when it happens on accident, I usually don't care if I update or not. For this reason, the pop-up is a hassle to me, and I would much prefer if the program just updated torrents regardless.
That being said, I cannot discount the possibility of some user wanting a very fine-grained control of when they update their podcasts, and so outright removing the prompt might affect some users negatively.

**Suggested fix**
Remove the "Look for new episodes?" prompt, or make it a toggleable option.

**Comment by y20k** [comment]
Hi @ScumbagDog

Thank you that you took the time to create good/actionable feature request!

But :-) the request came up in the past already (see #33). Pull to refresh is a gesture that can be triggered by accident - for example when swiping through the list of podcast. And since refreshing the podcast subscriptions can result a large network operation, I felt that a dialog is needed here.

Adding an update dialog toggle to the Settings screen is not a good solution for Escapepod, which is supposed to be/stay a very simple app.

**Comment by ScumbagDog** [comment]
That's very understandable, I just figured that checking for new episodes (not downloading them, because there is already a check for that) wouldn't be such a big operation to perform.

But ultimately, you probably know more about the network usage than I do. And yes, I agree, adding another setting in the options screen isn't a fantastic solution.
I just wanted to add my user feedback, in case it is useful to you :).

**Comment by y20k** [comment]
I just had a look: Updating 18 podcast feeds results in 16MB traffic. Not THAT much, but also not nothing.

**Comment by jmbmkn** [comment]
In this case isn't the dialogue there because the pull to refresh isn't reliable? Can the pull to refresh be adjusted to be less sensitive?

**Comment by y20k** [comment]
> Can the pull to refresh be adjusted to be less sensitive?

No. Not as far as I know. The pull to refresh gesture is a basic Android feature, that has so such parameters.
